### PR TITLE
Enhancement: option to show user for Tautulli and Emby/Jellyfin widgets

### PR DESCRIPTION
**Pull request by brikim**
## Proposed change

Added a new optional flag that can be added to the widget enableUser. If true the username will be added in parenthesis behind the title. Updated the documentation stating that this is an optional field and defaults to false.

TautulliEmbyJellyfinUsername
TautulliJellyfinEnableUserConfig

Closes #1233  #151 

## Type of change

- [ ] New service widget
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation only
- [ ] Other (please explain)

## Checklist:

- [x] If applicable, I have added corresponding documentation changes.
- [x] If applicable, I have reviewed the [feature](https://gethomepage.dev/latest/more/development/#new-feature-guidelines) and / or [service widget guidelines](https://gethomepage.dev/latest/more/development/#service-widget-guidelines).
- [x] I have checked that all code style checks pass using [pre-commit hooks](https://gethomepage.dev/latest/more/development/#code-formatting-with-pre-commit-hooks) and [linting checks](https://gethomepage.dev/latest/more/development/#code-linting).
- [x] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.

### CODE REVIEW

**Comment by brikim** [comment]
take 2 ... sorry about that I have no idea what was going on with my fork. Also if you would like me to remove the Emby/Jellyfin I can since the feature requests were just for Tautulli. I just added it to that widget for consistency. Thanks

File: src/utils/config/service-helpers.js [hunk 1]
```diff
@@ -517,6 +520,20 @@ export function cleanServiceGroups(groups) {
         if (["emby", "jellyfin"].includes(type)) {
           if (enableBlocks !== undefined) cleanedService.widget.enableBlocks = JSON.parse(enableBlocks);
           if (enableNowPlaying !== undefined) cleanedService.widget.enableNowPlaying = JSON.parse(enableNowPlaying);
+          if (enableUser !== undefined) { 
+            cleanedService.widget.enableUser = JSON.parse(enableUser);
+          } 
+          else { 
+            cleanedService.widget.enableUser = false;
+          }
```
Comment (shamoon) [inline src/utils/config/service-helpers.js:528]:
```suggestion
            cleanedService.widget.enableUser = !!JSON.parse(enableUser);
          }
```

File: src/utils/config/service-helpers.js [hunk 2]
```diff
@@ -517,6 +520,20 @@ export function cleanServiceGroups(groups) {
...
+        }
+        if (["tautulli"].includes(type)) {
+          if (enableUser !== undefined) { 
+            cleanedService.widget.enableUser = JSON.parse(enableUser);
+          } 
+          else { 
+            cleanedService.widget.enableUser = false;
+          }
```
Comment (shamoon) [inline src/utils/config/service-helpers.js:536]:
```suggestion
          if (enableUser !== undefined) { 
            cleanedService.widget.enableUser = !!JSON.parse(enableUser);
          }
```

File: src/widgets/emby/component.jsx [hunk 3]
```diff
@@ -215,6 +219,7 @@ export default function Component({ service }) {
 
   const enableBlocks = service.widget?.enableBlocks;
   const enableNowPlaying = service.widget?.enableNowPlaying ?? true;
+  const enableUser = service.widget?.enableUser ?? true;
```
Comment (shamoon) [inline src/widgets/emby/component.jsx:222]:
```suggestion
  const enableUser = !!service.widget?.enableUser;
```

This also currently makes it default to true

File: src/widgets/tautulli/component.jsx [hunk 4]
```diff
@@ -162,19 +168,21 @@ export default function Component({ service }) {
     );
   }
 
+  const enableUser = service.widget?.enableUser ?? true;
```
Comment (shamoon) [inline src/widgets/tautulli/component.jsx:171]:
```suggestion
  const enableUser = !!service.widget?.enableUser;
```

Same as above

**Review by shamoon — CHANGES_REQUESTED** [review]

**Comment by brikim** [comment]
I applied the comments and retested and everything is still working as expected. Sorry I think I pulled that from another example. I tested the cases with no label in my services file and it wasn't showing username before not sure how that was working.

**Review by shamoon — APPROVED** [review]
No worries, thanks

**Pull request merged**